runconfig/merge.go

```diff
@@ -20,6 +20,17 @@ func Merge(userConf, imageConf *Config) error {
 	if userConf.CpuShares == 0 {
 		userConf.CpuShares = imageConf.CpuShares
 	}
+
+	// FIXME: Config.Cpuset is obsoleted, will be removed.
+	if userConf.CpusetCpus != "" {
+		userConf.Cpuset = userConf.CpusetCpus
+	} else if userConf.Cpuset != "" {
+		userConf.CpusetCpus = userConf.Cpuset
+	} else {
+		userConf.Cpuset = imageConf.Cpuset
+		userConf.CpusetCpus = imageConf.CpusetCpus
+	}
+
```

Comment:
This is unnecessary: we don't want these fields to appear in the image (it's host dependent), so the merging behavior shouldn't be required.